### ISSUE TITLE
Passing the `-- --help` flag to a script should not present Composer's own help

### ISSUE THREAD

**Issue by johnbillion** [composer/composer#8165]
Passing the `-- --help` flag to a Composer script shows Composer's own help screen, instead of passing the flag through to the script as expected.

My `composer.json`:

```json
*snip*
"scripts": {
	"test": "vendor/bin/phpunit"
}
```

When I run this command:

```
composer test -- --help
```

I get the following output:

```
Usage:
  test [options] [--] [<args>]...

Arguments:
  args

Options:
...
```

And I expected this to happen:

I expected Composer to call PHPUnit with the `--help` flag passed to it so that I can see PHPUnit's help screen. Composer displays its own help screen instead.

**Comment by alcohol** [comment]
This looks like behaviour inherited from `symfony/console`: https://github.com/symfony/console/blob/cbcf4b5e233af15cd2bbd50dee1ccc9b7927dc12/Application.php#L159-L167

**Comment by johnbillion** [comment]
Thanks. It looks like the same problem also affects the shorthand `-h` along with the `--version` and `-V` flags in that case.

**Comment by Seldaek** [comment]
It's a known issue which we won't be able to fix until Composer v3 when we drop old PHP versions and can then upgrade symfony/console. Will just have to live with it until then I'm afraid.

**Comment by rubinlinux** [comment]
This appears to be resolved by composer 2.4.4 or earlier. It works as documented/expected in 2.4.4 and 2.5.1. 

'composer selfupdate' may be all that is needed if you are having this problem.
